packages/scss/src/components/clear/index.scss

```diff
@@ -36,7 +36,9 @@
 		.filterPill:not(.is-filled) &,
 		.textfield-input[disabled] &,
 		.textfield-input:not(.is-filled) & {
-			@include hidden;
+			.multiSelect:not(.is-clearable) & {
```

Comment:
This, shouldn't be nested, right?
<img width="866" height="103" alt="image" src="https://github.com/user-attachments/assets/2878a93d-c995-4c61-af19-5540ea291088" />